backend/chalice/api_server/vendor.in/psycopg2/__init__.py

```diff
@@ -8,8 +8,8 @@
 
 Homepage: http://initd.org/projects/psycopg2
 
-.. _PostgreSQL: http://www.postgresql.org/
-.. _Python: http://www.python.org/
+.. _PostgreSQL: https://www.postgresql.org/
```

Comment:
Everything in this directory is copy pasted from https://github.com/jkehler/awslambda-psycopg2/tree/master/psycopg2-3.7